reefer_monitoring/v1/reefer_v1.0.0-Beta-1.yaml

```diff
@@ -0,0 +1,524 @@
+openapi: 3.0.0
+x-stoplight:
```

Comment:
When you export from StopLight please uncheck the  "X-Extensions"
<img width="379" alt="image" src="https://github.com/dcsaorg/DCSA-OpenAPI/assets/3489388/700ff9fe-ac3c-4d56-b704-06779019ee99">
This removes these x-stoplight attributes